runtimes/pythonrt/runner/main.py

```diff
@@ -262,10 +262,10 @@ def Start(self, request: pb.runner.StartRequest, context: grpc.ServicerContext):
                 f"function {fn_name!r} not found",
             )
 
-        event = json.loads(request.event.data)
-
-        fix_http_body(event)
-        event = AttrDict(event)
+        inputs = json.loads(request.event.data)
+        fix_http_body(inputs)
+        inputs["data"] = AttrDict(inputs["data"])
+        event = Event(**inputs)
```

Comment:
```suggestion
        inputs = json.loads(request.event.data)
        fix_http_body(inputs)
        session_id, data = inputs["session_id"], AttrDict(inputs["data"])
        event = Event(session_id, data)
```
Don't tie the API layer `data` to the `Event` in Python. Things might change.

Reply:
Do you have an alternative? If data was not `AttrDict` I would have just done `Event(**event)`.

Reply:
ignore me, i see what you did there.